x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/transform/action/UpdateTransformAction.java

```diff
@@ -38,23 +41,36 @@ private UpdateTransformAction() {
         super(NAME, Response::new);
     }
 
-    public static class Request extends AcknowledgedRequest<Request> {
+    public static class Request extends BaseTasksRequest<Request> {
```

Comment:
~~Now this is a Tasks request it should override `public boolean match(Task task)` so that the transform ID in the request can be mapped to the transfrom task.~~

I scrolled down and see that this is handled in the action